docs/setting-up/client/mongodb.md

```diff
@@ -217,7 +217,8 @@ Use `pmm-admin` to add the database server as a service using one of these examp
 When successful, PMM Client will print `MongoDB Service added` with the service's ID and name. Use the `--environment` and `-custom-labels` options to set tags for the service to help identify them.
 
 !!! hint alert alert-success "Tips"
-    - When adding nodes of a sharded cluster, add each node separately using the `--cluster mycluster` option for the [MongoDB Cluster Summary](../../details/dashboards/dashboard-mongodb-cluster-summary.md) dashboard to populate correctly. Also use the `--replication-set` option to specify a replication set. Example: `--replication-set config` for your config servers; `--replication-set rs1` for your servers in the first replica set, `--replication-set rs2` for your servers in the second replica set, and so on.
+    - When adding nodes of a sharded cluster, add each node separately using the `--cluster mycluster` option for the [MongoDB Cluster Summary](../../details/dashboards/dashboard-mongodb-cluster-summary.md) dashboard to populate correctly. 
+    - You can also use the `--replication-set` option to specify a replication set, altough they are automatically detected. Example: `--replication-set config` for your config servers; `--replication-set rs1` for your servers in the first replica set, `--replication-set rs2` for your servers in the second replica set, and so on.
```

Comment:
> altough they are automatically detected

Are they? Most of dashboards keep working, because we just use another label. But for backups as far as I remember it's critical to pass that flag.

Reply:
well, we can derive the replication set name from metrics, so it should be auto detected. Otherwise the user could supply the wrong replication-set which would cause all sorts of problems.

Reply:
you are right, but it's not about can we or can't we. it's about do we or don't we. --replication-set generates label that is different from `set` label, so until we formalise and make autodetect I wouldn't mention that in documentation.

Reply:
I agree. To validate if it's really needed, I ran a quick test and backups seems to work fine even without providing the --replication-set flag. Do you have any example or remember under which conditions may the backup fail?

Reply:
No, that was just an assumption, I checked a code and seems like we use RS for PITR, but it's read from Mongo itself.